### DISCUSSION TITLE
Expand single attribute documentation

### DISCUSSION THREAD

**Discussion by ZeroPie** [pocketbase/pocketbase#4901] [Q&A] [answered]
https://github.com/pocketbase/pocketbase/issues/4900

@ganigeorgiev Thanks for your fast answer.

Maybe my lack of understanding/not reading the complete documentation caused this, but i don't know how i would have figured that out from the docs alone:

https://pocketbase.io/docs/working-with-relations#expanding-relations

Do you think it's worth adding it to the documentation?

Wish you a nice Saturday!

**Comment by ganigeorgiev** [comment, marked as answer]
The docs will be updated with the [refactoring](https://github.com/pocketbase/pocketbase/discussions/4355) and I'll consider adding an example with the `fields` query param.

In general you can also find details about all available list query parameters under the "API Preview" for your collections (List/Search > "Query parameters"  table) and/or in the common [Web API List docs](https://pocketbase.io/docs/api-records/#listsearch-records).
